tests/Mutator/AbstractMutatorTestCase.php

```diff
@@ -34,12 +34,14 @@
 
     public function doTest(string $inputCode, string $expectedCode = null)
     {
+        $inputCode = rtrim($inputCode, "\n");
```

Comment:
We trim any trailing new line for both the input and expected code, so we don't have to worry about that any more.